### PR TITLE
PXC-2405: wsrep_sst_auth - remove

### PR DESCRIPTION
**Pull request by borys-belinsky**

### CODE REVIEW

File: doc/source/add-node.rst [hunk 1]
```diff
@@ -15,17 +15,14 @@ it automatically joins the cluster and synchronizes with it.
...
-By default, |PXC| uses |PXB|_ for *State Snapshot Transfer* (:term:`SST`).
+By default, |PXC| uses |PXB|_ for :term:`State Snapshot Transfer <SST>`.
 This requires the following:
 
-* Set the :variable:`wsrep_sst_method`` variable to ``xtrabackup-v2`` and
-  provide SST user credentials with the :variable:`wsrep_sst_auth`` variable.
-
+* Set the :variable:`wsrep_sst_method`` variable to ``xtrabackup-v2``.
   For more information, see :ref:`configure`.
+* Create a user for SST on the initial node. For more information, see
```
Comment (kennt-percona) [inline doc/source/add-node.rst]: We do not need to "Create a user for SST on the initial node."  This was the user in the wsrep_sst_auth.

File: doc/source/manual/state_snapshot_transfer.rst [hunk 2]
```diff
@@ -84,8 +66,6 @@ Using mysqldump
 This method uses the standard :program:`mysqldump` utility
```
Comment (kennt-percona) [inline doc/source/manual/state_snapshot_transfer.rst]: This entire section for mysqldump should be removed also.  For 8.0, the mysqldump method has been removed.

Comment (borys-belinsky) [reply doc/source/manual/state_snapshot_transfer.rst]: Ready for review

Comment (borys-belinsky) [reply doc/source/manual/state_snapshot_transfer.rst]: Ready for review

File: doc/source/manual/state_snapshot_transfer.rst [hunk 3]
```diff
@@ -11,13 +11,12 @@ In order to be synchronized with the cluster,
 the new node has to receive data from a node
 that is already part of the cluster.
 
-There are three methods of SST available in |PXC|:
+There are two methods of SST available in |PXC|:
```
Comment (kennt-percona) [inline doc/source/manual/state_snapshot_transfer.rst]: rsync has also been removed from 8.0.  So there is only one sst method available, "xtrabackup".  Although you may want to create another PR to remove the rsync docs.

**Review by kennt-percona — APPROVED** [review]

**Pull request merged**